frb_dart/lib/src/basic.dart

```diff
@@ -11,6 +11,15 @@ import 'package:meta/meta.dart';
 
 final _instances = <Type>{};
 
+class PanicError extends FrbException {
```

Comment:
uber-nit: Shall it be `PanicException`, since it implements Frb"Exception" and Exception as well